rook/helmfile.d/upstream/library/Chart.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v2
```

Comment:
Can you explain why you need to add this chart when it is already packaged in with rook-ceph and rook-ceph-cluster?

Reply:
When looking upstream this looks correct :thinking:

Reply:
It's in the upstream github repo as a chart but it is packaged with both the cluster and operator chart as it's listed as a dependency.

Reply:
> Can you explain why you need to add this chart when it is already packaged in with rook-ceph and rook-ceph-cluster?



> When looking upstream this looks correct 🤔



> It's in the upstream github repo as a chart but it is packaged with both the cluster and operator chart as it's listed as a dependency in both of them.

It is a bit of a complicated situation in that version 14 introduces objects that haven't been used before. I will show how it all works to @robinAwallace tomorrow and we will see if any technological difficulties emerges.